### DISCUSSION TITLE
[Feature] Ability to modify/edit/delete face tags on pictures

### DISCUSSION THREAD

**Discussion by jkumeboshi** [immich-app/immich#3603] [Feature Request]
### The feature

Sometime Immich face recognition fails to recognize a face, so a picture is wrongly tagged with the wrong Person.

It would be nice to have following features (by priority):
1. Ability to remove a picture tag from a Person
2. Manually tag a picture with an existing Person
3. Re-trigger face recognition for a specified picture

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by mvivaldi** [comment]
In my opinion this is a very important feature to have (I have too many mislabeled persons in my instance!), but I think this is already in the upcoming features:
point 10 on https://github.com/immich-app/immich/discussions/2472
